detection_rules/etc/stack-schema-map.yaml

```diff
@@ -110,6 +110,6 @@
   endgame: "8.4.0"
 
 "8.16.0":
-  beats: "8.14.3"
+  beats: "8.15.0"
```

Comment:
Looks like 8.14.10 and 8.15.0 can have their beats version's updated as well.
Ref: https://github.com/elastic/beats/tags

Reply:
Done